receiveNew.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width">
+    <title>Peer-to-Peer Reciever</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <table class="display">
+        <tr>
+            <td class="title">Status:</td>
+            <td class="title">Messages:</td>
+        </tr>
+        <tr>
+            <td>
+                <div id="receiver-id" style="font-weight: bold;" title="Copy this ID to the input on send.html.">ID:
+                </div>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <div id="status" class="status"></div>
+            </td>
+            <td>
+                <div class="message" id="message"></div>
+            </td>
+        </tr>
+    </table>
+    <video width="400" controls></video>
+
+    <script src="https://unpkg.com/peerjs@1.3.1/dist/peerjs.min.js"></script>
+    <script type="text/javascript">
```

Comment:
as this is PoC, it doesn't really matter, but we should abstract js code in a separate file.

Reply:
done :D